src/Sensors/Sentinel1/FileIO/Metadata/Sentinel1Metadata.jl

```diff
@@ -5,7 +5,17 @@ include("MetadataUtil.jl")
 ##################### Constructors  ##################
 ######## Constructors for all the structures ########
 ######################################################
+function get_reference_time(meta_dict)::DateTime
+    start_time = meta_dict["product"]["adsHeader"]["startTime"]
+    reference_time = DateTime(start_time[1:19])
+    return reference_time
+end
 
+function parse_delta_time(time_string::String,reference_time::DateTime)
+    milliseconds = Dates.value( DateTime(time_string[1:23]) - reference_time) 
+    microseconds = parse(Int, time_string[24:26])
```

Comment:
same: indexing always the same?

Reply:
Yes. It is the date format used in the metadata file